02Lesson/01Project/index.html

```diff
@@ -1,9 +1,60 @@
 <!DOCTYPE html>
+<!--COMPLETED-->
 <html>
-<head>
-  <title></title>
-</head>
-<body>
+  <head>
+    <meta charset="utf-8">
+    <title>I Tech 4U</title>
+  </head>
+  <body>
+    <img src="http://geeks.eku.edu/sites/geeks.eku.edu/files/Images/cartoon-no-virus-hygienic-cute-sign-3090643.jpg" alt="virus">
+    <h1>
+      I Tech 4U
+    </h1>
+    <main>
+        <article>
+            Let us set it up for you!
+        </article>
+        <section>
+          Anything from routers to computer repair, we do it all!
+        </section>
+        <br>
+        <hr>
+      <div>
+        <h2>
+          Service with a smile :)
+        </h2>
+        <h3>
+          We firmly believe there are no stupid questions.
+        </h3>
+        <p>
+          <article>To completely service all your essential tech needs, our specialists are<!--save space. better way?-->
+            trained for any type of situation ranging from network setups, to inhome computer repair.
+          </article>
+          <h3>
+            Typical services offered:
+          </h3>
+          <p>
+            <ul>
+              <li>Network setup</li>
+              <li>Password reset</li>
+              <li>Slow running computer</li>
+              <li>Solutions to: "Why the f*!@ is this not working"</li>
+              <li>Computer repair/upgrades</li>
+            </ul>
 
-</body>
+        </div>
+        <div>
+          <h3>
+            Why choose us?
+          </h3>
+          <p>
```

Comment:
i keep getting the error code "at line 56, character 14No “p” element in scope but a “p” end tag seen." when i insert the closing </p> tag